tardis/io/tests/test_config_reader.py

```diff
@@ -68,3 +71,78 @@ def test_config_hdf(hdf_file_path, tardis_config_verysimple):
     actual = pd.read_hdf(hdf_file_path, key="/simulation/config")
     expected = expected.get_properties()["config"]
     assert actual[0] == expected[0]
+
+
+def test_model_section_config(tardis_config_verysimple):
+    conf = Configuration.from_config_dict(
```

Comment:
Please add a docstring describing what this test does

Reply:
Sure, I didn't know that we had to add docstrings for Tests too ... though I saw the `docstring_coverage` in the red :)